cypress/e2e/allMovies_spec.cy.js

```diff
@@ -0,0 +1,35 @@
+describe("Main Movie Page", () => {
```

Comment:
curious why the all movie is in this PR again? was main not pulled in beforehand?